public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -20,16 +20,16 @@ const OFFSET_X = 10;
 const OFFSET_Y = 32;
 
 export function InlineEdit({ onClose, id, scene }: Props) {
-  const root = scene.root.div!.getBoundingClientRect();
+  const root = scene.root.div?.getBoundingClientRect();
   const windowHeight = window.innerHeight;
   const windowWidth = window.innerWidth;
   const ref = useRef<HTMLDivElement>(null);
   const styles = useStyles2(getStyles);
   const inlineEditKey = 'inlineEditPanel' + id.toString();
 
   const defaultMeasurements = { width: 350, height: 400 };
-  const defaultX = root.x + root.width - defaultMeasurements.width - OFFSET_X;
-  const defaultY = root.y + OFFSET_Y;
+  const defaultX = root?.x ?? 0 + (root?.width ?? 0) - defaultMeasurements.width - OFFSET_X;
+  const defaultY = root?.y ?? 0 + OFFSET_Y;
```

Comment:
Not 💯 on these fallback values, @drew08t 

This change was necessary as we change out the root div when modifying inline editing / show advanced element types toggle

Reply:
For X, the default positioning is to place the editor slightly left of the right side of the panel. If root.x or root.width are undefined, I would place the editor at +OFFSET_X. Currently, this will place the editor off the screen, to the left.

Y looks good.